server/fleet/scripts.go

```diff
@@ -590,23 +592,27 @@ type BatchExecutionHost struct {
 }
 
 type BatchActivity struct {
-	ID              uint                       `db:"id"`
-	ScriptID        *uint                      `db:"script_id"`
-	ExecutionID     string                     `db:"execution_id"`
-	UserID          *uint                      `db:"user_id"`
-	JobID           *uint                      `db:"job_id"`
-	Status          BatchExecutionStatus       `db:"status"`
-	ActivityType    BatchExecutionActivityType `db:"activity_type"`
-	NumTargeted     *uint                      `db:"num_targeted"`
-	NumPending      *uint                      `db:"num_pending"`
-	NumRan          *uint                      `db:"num_ran"`
-	NumErrored      *uint                      `db:"num_errored"`
-	NumIncompatible *uint                      `db:"num_incompatible"`
-	NumCanceled     *uint                      `db:"num_canceled"`
-	CreatedAt       time.Time                  `db:"created_at"`
-	UpdatedAt       time.Time                  `db:"updated_at"`
-	CompletedAt     *time.Time                 `db:"completed_at"`
-	CanceledAt      *time.Time                 `db:"canceled_at"`
+	ID               uint                       `json:"id" db:"id"`
+	BatchExecutionID string                     `json:"batch_execution_id" db:"execution_id"`
+	UserID           *uint                      `json:"user_id" db:"user_id"`
+	JobID            *uint                      `json:"-" db:"job_id"`
+	ActivityType     BatchExecutionActivityType `json:"-" db:"activity_type"`
+	ScriptID         uint                       `json:"script_id" db:"script_id"`
```

Comment:
If we suspect we'll use the BatchActivity for other uses in the future, should the ScriptID be a pointer?

Reply:
good call, will update

Reply:
Done!